### DISCUSSION TITLE
Welcome to ChaosCE Discussions!

### DISCUSSION THREAD

**Discussion by spontificae** [spontificae/ChaosCE#1] [Announcements]
What would you like to see?

**Comment by aacquesta** [comment]
Hello.  I saw your pok file, and I enjoy very much your approach.(specially the walls, and lists randomization). I have been a Chaos fan since 1985. I've played with school fiends as a teenager, with my role play mates, college pals, and recently with my son and his friends.   forty years playing chaos! (Apologize my english, I'm from Argentina) I'm interested in the exapanded version, with 3 extra blocks arround the board. Is it available? I send you a redesign of the loading screen, for a T-shirt. I did it for the son's 13th birthday.  Tx a lot.   Alejandro.
